from django.core.exceptions import ValidationError
import re

def validate_diff_level(diff_level):
    pass